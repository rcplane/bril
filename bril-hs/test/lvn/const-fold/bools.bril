@main {
  tru: bool = const true;
  fls: bool = const false;

  disj: bool = or tru fls;
  conj: bool = and tru fls;
  neg: bool = not tru;

  print disj conj neg;
}
