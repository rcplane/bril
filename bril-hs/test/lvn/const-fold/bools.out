@main {
  disj: bool = const true;
  conj: bool = const false;
  neg: bool = const false;
  print disj conj neg;
}
